requirements.txt

```
@@ -7,7 +7,7 @@ opencv-python-headless
 # We cannot use constructs like these when passing directly to setuptools.
 # So instead of explicating imgaug here, let's rely on mask-rcnn for it.
 # imgaug --no-binary imgaug # workaround imgaug#473 (opencv-python/headless)
-pycocotools >= 2.0.2
+pycocotools==2.0.0
```

I suppose this was by accident, too?
```suggestion
pycocotools >= 2.0.2
```